server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -1142,8 +1142,10 @@ boolean canSuspendTablets() {
     }
 
     try {
-      final AgeOffStore<Manager> store = new AgeOffStore<>(new org.apache.accumulo.fate.ZooStore<>(
-          getZooKeeperRoot() + Constants.ZFATE, context.getZooReaderWriter()), 1000 * 60 * 60 * 8);
+      final AgeOffStore<Manager> store = new AgeOffStore<>(
+          new org.apache.accumulo.fate.ZooStore<>(getZooKeeperRoot() + Constants.ZFATE,
+              context.getZooReaderWriter()),
+          1000 * 60 * 60 * 8, System::currentTimeMillis);
```

Comment:
Not super important here, but when I see stuff like these multipliers, I find that it's a great opportunity to swap out with something more clear, like:
```suggestion
          TimeUnit.HOURS.toMillis(8), System::currentTimeMillis);
```